code/modules/mob/language.dm

```diff
@@ -406,6 +406,15 @@
 	key = "3"
 	syllables = list ("gra","ba","ba","breh","bra","rah","dur","ra","ro","gro","go","ber","bar","geh","heh", "gra")
 
+/datum/language/sign
```

Comment:
This needs to be set so that it does not work on radio.

Reply:
Understood, I'll get that sorted :)